app/uk/gov/hmrc/simpleapideploymentstubs/models/UpdateMetadata.scala

```diff
@@ -20,7 +20,10 @@ import play.api.libs.json.{Format, Json}
 
 case class UpdateMetadata(
   description: String,
-  status: String
+  status: String,
+  domain: Option[String] = None,
+  subdomain: Option[String] = None,
+  backends: Seq[String] = Seq.empty
 )
```

Comment:
What actually sets these values? Do we need to copy values from an input model? I think that's done in an apply() method.

Reply:
No ignore that, I though I was looking at api-hub-applications. Ho hum. Friday afternoon.